code/datums/diseases/allergy.dm

```diff
@@ -0,0 +1,54 @@
+/datum/disease/anaphylactic_shock
+	name = "Anaphylactic Shock"
+	form = "Shock"
+	max_stages = 4
+	spread_flags = NON_CONTAGIOUS
+	cure_text = "Epinephrine"
+	cures = list("epinephrine")
+	viable_mobtypes = list(/mob/living/carbon/human)
+	desc = "A severe allergic reaction. Untreated can prove lethal very quickly."
+	severity = DANGEROUS
+	cure_chance = 50
+	stage_prob = 10//advances FAST
+
+/datum/disease/anaphylactic_shock/stage_act()
+	..()
+
+	if(!ishuman(affected_mob))
+		return
+
+	var/mob/living/carbon/human/H = affected_mob
+
+	if(NOBREATH in H.dna.species.species_traits)
+		cure()
+
+	switch(stage)
+		if(1)
+			if(prob(8) && H.stat == CONSCIOUS)
+				var/list/oh_dear = list("Your chest hurts.", "You feel a cold sweat form.", "Your stomach violently rumbles!", "Your stomach hurts.")
```

Comment:
Why not declare these lists on the datum itself rather than re-declaring it each time the prob procs in this switch...?

Also ha ha variable names... can we give these something logical please.